### PR TITLE
Add names to commands and show on dashboard

### PR DESCRIPTION
**Pull request by jwt388**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -40,6 +41,8 @@ public void robotInit() {
     this.robotContainer = new RobotContainer();
 
     datalog.dataLogRobotContainerInit(this.robotContainer);
+
+    SmartDashboard.putData(CommandScheduler.getInstance());
```
Comment (blu28) [inline src/main/java/frc/robot/Robot.java:45]: Is there no way to put this into dataLogRobotContainerInit? I'd like to isolate most of the logging in datalogging.java if possible, and putData calls into RobotContainer.java.

**Review by blu28 — CHANGES_REQUESTED** [review]
Ca we move the new putdata call into datalogging.java?

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (jwt388) [reply src/main/java/frc/robot/Robot.java:45]: That should be fine. I’ll try it this evening and update. I think I just followed the WIPLib example.

**Comment by jwt388** [comment]
It looks like dataLogRobotContainerInit is the best spot for this, so moved logging of CommandScheduler there. I also moved logging of the drive and arm subsystem commands to the same location in the code and displayed them on the Shuffleboard Command tab so they are displayed by default and shown together.  Issue 76 branch updated.

**Review by blu28 — APPROVED** [review]

**Pull request merged**